scout/models/phenotype_term.py

```diff
@@ -27,49 +27,3 @@ class HpoTerm(BaseModel):
     def set_hpo_number(self) -> "HpoTerm":
         self.hpo_number = int(self.hpo_id.split(":")[1])
         return self
-
-
-# Disease terms represent diseases collected from omim, orphanet and decipher.
-# Collected from OMIM
-class DiseaseTerm(dict):
```

Comment:
See if you can't find a way to preserve the description of the db mapping object if you are not doing two different ones. Doc string on the Pydantic DiseaseTermLoader-object might be a little confusing? Just a non-functional `dict`in `scout/models/disease_term.py` perhaps?